packages/template-ui/src/main.js

```diff
@@ -1,5 +1,6 @@
 import Vue from 'vue';
 import { BootstrapVue, IconsPlugin } from 'bootstrap-vue';
+import { i18nSetup } from 'kolibri.utils.i18n';
```

Comment:
This fails as it requirer `kolibri` as dependency of the template-ui. I guess you are testing this on top of the other PR?

Here is my alternative proposal #790 . This is what I was meaning about passing the translations through the custom presentation API.

Reply:
> This fails as it requires `kolibri` as dependency of the template-ui. I guess you are testing this on top of the other PR?

I don’t think I was testing it on top of the other PR, otherwise the branch would be based on that one. The CI seems happy with it as well.

> Here is my alternative proposal #790 . This is what I was meaning about passing the translations through the custom presentation API.

That looks fine to me, though I don’t really know what I’m talking about. Are you sure we don’t need the other things that `i18nSetup()` does, though? It sets up language and font settings, `Intl` polyfills, and imports locale data.